src/index.css

```diff
@@ -124,12 +124,19 @@ html[data-theme="rosebox"] {
   --success-dark: #00915d;
 }
 
+/* Fix SUID weird behavior with dark theme */
+/* It generates weird lineal-gradient to use as bg which overrides bgcolor */
+/* Ignores bgcolor = ignores custom theme */
+[class*="css-"] {
```

Comment:
tbh this is kinda husk.
is it really needed?
please atleast use ^= if this can't be done in a better way.

Reply:
okay whatever

Reply:
![image](https://github.com/unfinishedprojs/demo-front/assets/68553709/5cf8b040-ab9d-4d90-97f4-66daccf8ed52)
both normal background and bg-image are created. Why? no idea